cloud/pkg/cloudstream/iptables/iptables.go

```diff
@@ -46,12 +52,15 @@ const (
 	tunnelPortChain utiliptables.Chain = "TUNNEL-PORT"
 )
 
-var iptablesJumpChains = []iptablesJumpChain{
-	{utiliptables.TableNAT, tunnelPortChain, utiliptables.ChainOutput, "kubeedge tunnel port", nil},
-	{utiliptables.TableNAT, tunnelPortChain, utiliptables.ChainPrerouting, "kubeedge tunnel port", nil},
-}
+var (
+	iptablesJumpChains = []iptablesJumpChain{
+		{utiliptables.TableNAT, tunnelPortChain, utiliptables.ChainOutput, "kubeedge tunnel port", nil},
+		{utiliptables.TableNAT, tunnelPortChain, utiliptables.ChainPrerouting, "kubeedge tunnel port", nil},
+	}
+	kubeClient *kubernetes.Clientset
+)
 
-func NewIptablesManager(streamPort int) *Manager {
+func NewIptablesManager(config *cloudcoreConfig.KubeAPIConfig, streamPort int) *Manager {
```

Comment:
In future, we can use native apiserver config here :) And now it has no ci test, let's merge the helm test ASAP.

Reply:
> In future, we can use native apiserver config here :) And now it has no ci test, let's merge the helm test ASAP.

Yeah.